src/content/en/2022/capabilities.md

```diff
@@ -16,7 +16,372 @@ featured_stat_2: TODO
 featured_stat_label_2: TODO
 featured_stat_3: TODO
 featured_stat_label_3: TODO
-unedited: true
 ---
 
-## TODO
+## Introduction
+
+Compelling web experiences aren't limited to basic browser capabilities; they can take advantage of their underlying operating system. Web platform APIs expose these capabilities that are the foundation for [Progressive Web Apps (PWA)](./pwa_)—web applications capable of providing high-quality experiences found like platform-specific apps.
```

Comment:
```suggestion
Compelling web experiences aren't limited to basic browser capabilities; they can take advantage of their underlying operating system. Web platform APIs expose these capabilities that are the foundation for [Progressive Web Apps (PWA)](./pwa_)—web applications capable of providing high-quality experiences like platform-specific apps.
```

I'm not a native speaker, but this looks wrong to me.

Reply:
Nope, that's me having a brain fart. Nice catch.